src/_includes/components/ai-expert-modal.njk

```diff
@@ -1072,23 +1083,24 @@ document.addEventListener('DOMContentLoaded', function() {
                     </div>
                 </li>`;
             });
-            
+
             html += '</ol>';
             html += '</div>';
         }
-        
+
         // Node Packages
         if (richAnswer.nodePackages && richAnswer.nodePackages.length > 0) {
             html += '<div class="mb-4">';
             html += '<h4 class="text-base font-medium text-gray-900 mb-3">Required Node Packages</h4>';
             html += '<div class="grid grid-cols-1 md:grid-cols-2 gap-2">';
-            
+
             richAnswer.nodePackages.forEach(pkg => {
                 const nodeRedUrl = `https://flows.nodered.org/node/${pkg.name}`;
```

Comment:
As an aside, I think we should favour our certified nodes first, and keep the user locally on our website

Reply:
Perhaps we need something similar to the flows library for private nodes?